src/db/sql/sqlUpdateBuilder.ts

```diff
@@ -0,0 +1,47 @@
+import { Objects } from '@openforis/arena-core'
+import { SqlBuilder } from './sqlBuilder'
+import { Table } from '../table'
+import { Column } from '../column'
+
+export class SqlUpdateBuilder extends SqlBuilder {
+  private _update: Table[] = []
+  private _set: string[] = []
+  private _where: string[] = []
+  private _returning: Column[] = []
+
+  update(...table: Table[]): this {
+    this._update.push(...table)
+    return this
+  }
+
+  set(...tables: string[]): this {
+    this._set.push(...tables)
+    return this
+  }
```

Comment:
should the set have a signature like: (column:Column|string, value)? and then construct the set field=value in the build method?